graphql/typeDefs.ts

```diff
@@ -170,8 +170,12 @@ export default gql`
   type Star {
     id: String!
     studentId: Int
+    studentUsername: String
+    studentName: String
     mentorId: Int
     lessonId: Int
+    lessonTitle: String
+    lessonOrder: Int
```

Comment:
It looks like you use `lessonOrder` as a string in the `helpers/controllers/userInfoController.ts` file, but here you are declaring it as an integer. According to graphql specs, graphql will coerce data types as needed into the one defined in the schema in order to `uphold the contract the scalar type describes`

See here, and search `result coercion` for more information:
https://spec.graphql.org/June2018/

Reply:
thanks, fixed!

also just added id to userInfo/starsReceived